client/src/config.ts

```diff
@@ -40,3 +21,66 @@ export const ipfsGatewayUri =
     process.env.REACT_APP_IPFS_GATEWAY_URI || 'https://gateway.ipfs.io'
 export const ipfsNodeUri =
     process.env.REACT_APP_IPFS_NODE_URI || 'https://ipfs.infura.io:5001'
+
+//
+// OCEAN REMOTE CONNECTIONS
+//
+export const CONNECTIONS = {
+    pacific: {
+        nodeUri:
+            process.env.REACT_APP_NODE_URI ||
+            'https://pacific.oceanprotocol.com',
+        aquariusUri:
+            process.env.REACT_APP_AQUARIUS_URI ||
+            'https://aquarius.commons.oceanprotocol.com',
+        brizoUri:
+            process.env.REACT_APP_BRIZO_URI ||
+            'https://brizo.commons.oceanprotocol.com',
+        brizoAddress:
+            process.env.REACT_APP_BRIZO_ADDRESS ||
+            '0x008c25ed3594e094db4592f4115d5fa74c4f41ea',
+        secretStoreUri:
+            process.env.REACT_APP_SECRET_STORE_URI ||
+            'https://secret-store.oceanprotocol.com',
+        faucetUri:
+            process.env.REACT_APP_FAUCET_URI ||
+            'https://faucet.oceanprotocol.com'
+    },
+    nile: {
+        nodeUri: process.env.REACT_APP_NODE_URI || 'https://nile.dev-ocean.com',
+        aquariusUri:
+            process.env.REACT_APP_AQUARIUS_URI ||
+            'https://aquarius.nile.dev-ocean.com',
+        brizoUri:
+            process.env.REACT_APP_BRIZO_URI ||
+            'https://brizo.nile.dev-ocean.com',
+        brizoAddress:
+            process.env.REACT_APP_BRIZO_ADDRESS ||
+            '0x4aaab179035dc57b35e2ce066919048686f82972',
+        secretStoreUri:
+            process.env.REACT_APP_SECRET_STORE_URI ||
+            'https://secret-store.nile.dev-ocean.com',
+        faucetUri:
+            process.env.REACT_APP_FAUCET_URI ||
+            'https://faucet.nile.dev-ocean.com'
+    },
+    duero: {
+        nodeUri:
+            process.env.REACT_APP_NODE_URI || 'https://duero.dev-ocean.com',
+        aquariusUri:
+            process.env.REACT_APP_AQUARIUS_URI ||
+            'https://aquarius.duero.dev-ocean.com',
+        brizoUri:
+            process.env.REACT_APP_BRIZO_URI ||
+            'https://brizo.duero.dev-ocean.com',
+        brizoAddress:
+            process.env.REACT_APP_BRIZO_ADDRESS ||
+            '0x9d4ed58293f71122ad6a733c1603927a150735d0',
+        secretStoreUri:
+            process.env.REACT_APP_SECRET_STORE_URI ||
+            'https://secret-store.duero.dev-ocean.com',
+        faucetUri:
+            process.env.REACT_APP_FAUCET_URI ||
+            'https://faucet.duero.dev-ocean.com'
+    }
```

Comment:
for my Spree note in the comment below,I guess we need to add all the `spree` localhost connections here

Reply:
Handled the case with **spree** in config.ts based on `REACT_APP_OCEAN_NETWORK` from .env.local